test/functional/testlib.bash

```diff
@@ -1510,3 +1510,30 @@ assert_not_equal() {
 	fi
 
 }
+
+assert_files_equal() {
+
+	local val1=$1
+	local val2=$2
+	validate_item "$val1"
+	validate_item "$val2"
+
+	diff -q "$val1" "$val2"
+
+}
+
+assert_files_not_equal() {
+
+	local val1=$1
+	local val2=$2
+	validate_item "$val1"
+	validate_item "$val2"
+
+	if diff -q "$val1" "$val2" > /dev/null; then
+		echo "Files $val1 and $val2 are equal"
```

Comment:
should this library function actually print anything or should the `return 1` be enough?

Reply:
assert_files_equal() uses `diff -q` which will return a text like this'd a return code of 1 when files are not equal:
"Files tmp1 and tmp2 differ"

So by printing "Files $val1 and $val2 are equal" in function assert_files_not_equal() I'm attempting to being consistent in both functions.

Definitevely returning 1 is the most important thing, but I don't see why some extra info would hurt.